test/permission.validation.js

```diff
@@ -11,31 +11,31 @@ const {
 
 const runTests = (test, ctx) => {
   const testsDir = path.join(__dirname, 'permissions');
-
-  fs.readdir(testsDir, (err, testFiles) => {
-    if (err) {
-      console.error(
-        `Cannot read tests' directory, skipping permission validation tests.`
-      );
-      console.error(err);
-      test.end();
-      return;
-    }
-
-    for (const testFile of testFiles) {
-      const testFilePath = path.join(testsDir, testFile);
-      const subtest = require(testFilePath);
-      subtest(test, ctx);
-    }
-  });
+  fs.readdir(
+    testsDir,
+    test.cbFail(
+      `Cannot read tests' directory, skipping permission validation tests`,
```

Comment:
`cbFail` would fail the test in case of error. If that's the intent then the message needs changing.